metakb/deltas/delta.py

```diff
@@ -1,119 +1,124 @@
-"""A module for computing CIViC deltas."""
+"""A module for computing deltas."""
 from metakb import PROJECT_ROOT
 import json
 import logging
 from jsondiff import diff
 from datetime import date
-import pkg_resources
-from metakb.harvesters import CIViC
+from metakb.harvesters import CIViC, MOAlmanac
 
```

Comment:
Adding a dict here to reference classes by string can be used to reduce complexity in lines 50-55.

```suggestion
HARVESTER_CLASS = {
    'civic': CIViC,
    'moa': MOAlmanac
}
```

Reply:
This could even be automatically generated through class inspection, and/or defined and imported from the harvesters package __init__.py.